app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+
+  def create
+    @company = Company.find(params[:company_id])
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+    if @comment.save
+      redirect_to company_job_path(@company, @comment.job)
+    else
+      redirect_to new_company_job_comment_path
```

Comment:
we probably want to re render the view that had that form on it rather than redirecting so we don't lose the data the user tried to enter